velox/functions/lib/Re2Functions.cpp

```diff
@@ -2154,6 +2201,14 @@ std::shared_ptr<exec::VectorFunction> makeLike(
 
   PatternMetadata patternMetadata = PatternMetadata::generic();
   try {
+    // Fast path for substrings search.
+    auto substrings =
```

Comment:
After picking this PR, Gluten will be a unit test that fails.

`Incorrect evaluation: ///__ LIKE %//%/% ESCAPE '/', actual: true, expected: false `

It seems that this implementation did not take into account the ESCAPE. In this example, the last slash (/) is the ESCAPE, which causes the last percent (%) to be a regular character, not a wildcard. Therefore, the result should be false, but Velox returns true.
@skadilover @mbasmanova @Yuhta @xiaoxmeng Do you have any input?

Reply:
So sorry for the mistake, I mis-understand the code here: 
```
    BaseVector* escape = inputArgs[2].constantValue.get();
    if (!escape) {
      return std::make_shared<LikeGeneric>();
    }
```

and the ut just skiped by '#'
```
  testLike("a%c", "%#%%", '#', true);
  testLike("%cd", "%#%%", '#', true);
```

fix it here : https://github.com/facebookincubator/velox/pull/11091